src/shogun/multiclass/LaRank.h

```diff
@@ -485,6 +485,8 @@ namespace shogun
 
 			/// progess output
 			int32_t step;
+                        
+                        static const int32_t MAX_ITERATION = 1000;
```

Comment:
Also, something more well informed. How many iterations does this usually take until convergence? What does the literature say here?